calico-cloud/get-started/system-requirements.mdx

```diff
@@ -1,62 +1,50 @@
-| **CNIs**                    | - Calico CNI v3.20 to v3.27<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI |
-| **Browsers for Manager UI** | {{prodname}} supports the latest version and "latest minus one" versions of the following browsers:<br/><br/>- Chrome<br/>- Safari<br/>- Firefox |
+Calico Cloud works with Kubernetes on self-provisioned infrastructure and on managed Kubernetes distributions.
+To use Calico Cloud for both networking and network policy, your cluster must have Calico Open Source installed before you connect to Calico Cloud.
+For most managed distributions, you can use the provider's CNI for networking and use Calico Cloud for network policy.
 
-### Step 2: Verify cluster is not managed by a Kubernetes reconciler
+| Distribution | Supported CNIs |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
+| Amazon Elastic Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Amazon VPC CNI |
+| Azure Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Azure CNI |
+| Google Kubernetes Engine | - Calico Open Source 3.26 and 3.27<br/> - GKE CNI |
+| Rancher Kubernetes Engine 2 | - Calico Open Source 3.26 and 3.27 |
 
-{{prodname}} cannot be installed on clusters that are managed by any kind of Kubernetes reconciler (for example, Addon-manager). To verify, look for an annotation called `addonmanager.kubernetes.io/mode` on either of the following resources. (The resources may not exist).
 
-- `tigera-operator` deployment in the `tigera-operator` namespace (Installation on AKS may still work if the mode is Reconcile, see [AKS](aks.mdx#verify-addonmanager-is-compatible) for an additional check.)
-- `calico-node` daemonset in the `kube-system` namespace
+## Kubernetes versions
 
-If the following command finds addonmanager on either of the resources, then Addon-manager is being used. Find a different cluster to use.
+Your Kubernetes distribution must be based on one of the following Kubernetes versions:
+* Kubernetes 1.29
+* Kubernetes 1.28
+* Kubernetes 1.27
+* Kubernetes 1.26
 
-```bash
-kubectl get <type> -n <namespace> <resource-name> -o yaml | grep ' addonmanager.kubernetes.io/mode:'
-```
+## Architectures
 
-### Step 3: Prepare your infrastructure to connect to {{prodname}}
+Calico Cloud can be installed on nodes based on the following chip architectures:
+* x86-64
+* ARM64
 
-Pods running in your Kubernetes cluster must allow outbound traffic to the following endpoints:
+## Browser support for the Manager UI web console
 
-- `https://installer.calicocloud.io:443/*`
-- `https://www.calicocloud.io:443/api/*`
-- `https://client-auth.calicocloud.io:443/*`
-- TCP to `<management-cluster>.calicocloud.io:9000`
+To access the Manager UI web console, you can use latest two versions of the following web browsers:
+* Chrome
+* Firefox
```

Comment:
IIUC we do not test in Firefox at this time but there is work in progress to get this capability added to our automated tests.